### PR TITLE
HWKALERTS-252 Fix wrong nanoseconds parsed time

### PR DESCRIPTION
**Pull request by lucasponce**
HWKALERTS-251 Fix wrong dot character on Elasticsearch alerter
HWKALERTS-247 Add an Elasticsearch intergation example

Please @jshaughn, could you review ?

This would be necessary for a potential 1.6.x. and metrics 0.26.x.

### CODE REVIEW

File: hawkular-alerters/hawkular-elasticsearch-alerter/hawkular-elasticsearch-alerter-lib/src/main/java/org/hawkular/alerter/elasticsearch/ElasticsearchAlerter.java [hunk 1]
```diff
@@ -69,7 +69,7 @@
  *   Documents fetched from Elasticsearch need a date field to indicate the timestamp.
  *   This timestamp will be used in the queries to fetch documents in interval basis.
  *
- *   If there is not defined a specific pattern under the trigger.context["timestamp.pattern"] it will follow the
+ *   If there is not defined a specific pattern under the trigger.context["timestamp_pattern"] it will follow the
```
Comment (jshaughn) [inline hawkular-alerters/hawkular-elasticsearch-alerter/hawkular-elasticsearch-alerter-lib/src/main/java/org/hawkular/alerter/elasticsearch/ElasticsearchAlerter.java]: Why the change from the dot to the underscore?  I think ES does not allow a dot but this is an hAlerting contect name, no?

Comment (lucasponce) [inline hawkular-alerters/hawkular-elasticsearch-alerter/hawkular-elasticsearch-alerter-lib/src/main/java/org/hawkular/alerter/elasticsearch/ElasticsearchAlerter.java]: Because Elasticsearch 2.x version doesn't accept dot as part of field name.
And when storing a full alert, it might fail with context with dots.
It is fixed in upper versions but 2.x is the target used for OS deployments, so that's motivation.

**Comment by jshaughn** [comment]
Thanks, Lucas!
